docs/developer/index.md

```diff
@@ -1,13 +1,11 @@
-# Developer documentation
+# Development
 
 ```{include} ../../CONTRIBUTING.md
 ```
 
-## Table of contents
-
 ```{toctree}
 ---
-maxdepth: 2
+hidden:
 ---
```

Comment:
Why deviate from the common template?

Reply:
I thought we were allowed to try things out and if we like them we can propagate them to the template?

Reply:
Fair enough, I put it back in :+1:

Reply:
I think you edited the wrong file in your latest commit. My comment was about the dev docs, not the general docs.

Reply:
oops...